versioned_docs/version-2.0/concepts/admin/admin.mdx

```diff
@@ -0,0 +1,45 @@
+# Admin
```

Comment:
I think one operational piece we'll want to include somewhere is info on user deletion (org-level) and user removal (workspace level). Likely important to have all of the side effects laid out clearly. It's less conceptual so may belong elsewhere but wanted to flag.